migrations/Version20210305153851.php

```diff
@@ -0,0 +1,86 @@
+        $this->addSql('CREATE TABLE aid_business_activity_area (aid_id UUID NOT NULL, business_activity_area_id INT NOT NULL, PRIMARY KEY(aid_id, business_activity_area_id))');
+        $this->addSql('CREATE INDEX IDX_6C9B5ACECB0C1416 ON aid_business_activity_area (aid_id)');
+        $this->addSql('CREATE INDEX IDX_6C9B5ACE3087BC3B ON aid_business_activity_area (business_activity_area_id)');
+        $this->addSql('COMMENT ON COLUMN aid_business_activity_area.aid_id IS \'(DC2Type:uuid)\'');
+        $this->addSql('CREATE TABLE business_activity_areas (id INT NOT NULL, name VARCHAR(255) NOT NULL, slug VARCHAR(255) NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE TABLE environmental_actions (id INT NOT NULL, name VARCHAR(255) NOT NULL, slug VARCHAR(255) NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE TABLE environmental_topics (id INT NOT NULL, name VARCHAR(255) NOT NULL, slug VARCHAR(255) NOT NULL, PRIMARY KEY(id))');
+        $this->addSql('CREATE TABLE funders (id INT NOT NULL, name VARCHAR(255) NOT NULL, website VARCHAR(255) DEFAULT NULL, PRIMARY KEY(id))');
+        $this->addSql('ALTER TABLE aids ADD CONSTRAINT FK_2D4531F0C447CBA FOREIGN KEY (aid_advisor_id) REFERENCES aid_advisors (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aids ADD CONSTRAINT FK_2D4531F06CC88588 FOREIGN KEY (funder_id) REFERENCES funders (id) NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aid_environmental_action ADD CONSTRAINT FK_4784D526CB0C1416 FOREIGN KEY (aid_id) REFERENCES aids (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aid_environmental_action ADD CONSTRAINT FK_4784D526E7205469 FOREIGN KEY (environmental_action_id) REFERENCES environmental_actions (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aid_environmental_topic ADD CONSTRAINT FK_D5196A1BCB0C1416 FOREIGN KEY (aid_id) REFERENCES aids (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aid_environmental_topic ADD CONSTRAINT FK_D5196A1B67ABD4B6 FOREIGN KEY (environmental_topic_id) REFERENCES environmental_topics (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aid_business_activity_area ADD CONSTRAINT FK_6C9B5ACECB0C1416 FOREIGN KEY (aid_id) REFERENCES aids (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+        $this->addSql('ALTER TABLE aid_business_activity_area ADD CONSTRAINT FK_6C9B5ACE3087BC3B FOREIGN KEY (business_activity_area_id) REFERENCES business_activity_areas (id) ON DELETE CASCADE NOT DEFERRABLE INITIALLY IMMEDIATE');
+    }
+
+    public function down(Schema $schema) : void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('CREATE SCHEMA public');
```

Comment:
You should always remove this line in migrations, that's a glitch from Doctrine Migrations but this down method wouldn't work if you keep it.